backend/scripts/seed-local.sql

```
@@ -103,12 +106,12 @@ WHERE NOT EXISTS (SELECT 1 FROM companies WHERE id = 1);
 -- 1% を company_admin にして権限分岐のあるクエリも実データで踏めるようにする。
 -- ロールは role_id が正（FRESTYLE-311）。OIDC subject は user_oidc_identities に持つ。
 -- password_hash は全員 'password' の bcrypt（ローカルのパスワードログイン用・FRESTYLE-311 PR2）。
-INSERT INTO users (id, email, name, company_id, role_id, password_hash, is_active, created_at, updated_at)
+INSERT INTO users (id, email, name, workspace_id, role_id, password_hash, is_active, created_at, updated_at)
 SELECT
   1000000 + i,
   'seed' || i || '@example.test',
   'シード利用者' || i,
-  1,
+  (SELECT workspace_id FROM companies WHERE id = 1),
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)